lib/matplotlib/_mathtext.py

```diff
@@ -2144,6 +2149,9 @@ def symbol(self, s, loc, toks):
     def unknown_symbol(self, s, loc, toks):
         raise ParseFatalException(s, loc, f"Unknown symbol: {toks['name']}")
 
+    def unclosed_group(self, s, loc, toks):
+        raise ParseFatalException(s, len(s), "Expected '}'")
```

Comment:
Here is the other, identical, method. Should only require one and I think this is the one.

Reply:
Yup, I missed that. Thanks!